.storybook/test-runner.ts

```diff
@@ -0,0 +1,33 @@
+// .storybook/test-runner.ts
+
+import type { TestRunnerConfig } from '@storybook/test-runner';
+
+import { injectAxe, checkA11y, configureAxe } from 'axe-playwright';
+import { getStoryContext } from '@storybook/test-runner';
+
+/*
+ * See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
+ * to learn more about the test-runner hooks API.
+ */
+const a11yConfig: TestRunnerConfig = {
+  async preRender(page) {
+    await injectAxe(page);
+  },
+  async postRender(page, context) {
+    const storyContext = await getStoryContext(page, context);
+    // Apply story-level a11y rules
+    await configureAxe(page, {
+      rules: storyContext.parameters?.a11y?.config?.rules,
+    });
+
+    // Enable this to start having Accessibility checks in our tests!
```

Comment:
Enable this in a different story when we want to fix all the accessibility errors. there is not a warning type option i could find.

made: https://bugster.forgerock.org/jira/browse/SDKS-2698